File: jobs/2026-05-26__11-38-52/concurrency_limit_ts__rKQLHu9/artifacts/user/myproject/src/worker.ts
import { Hatchet } from '@hatchet-dev/typescript-sdk';
import { processOrder } from './task.js';

const hatchet = Hatchet.init();

async function main() {
  const worker = await hatchet.worker('order-worker');
  await worker.registerWorkflow(processOrder);
  worker.start();
  console.log('Worker started');
}

main().catch(console.error);

File: jobs/2026-05-26__10-49-31/bulk_run_py__62Qr5Jq/artifacts/user/myproject/worker.py
from hatchet_sdk import Hatchet
from task import hatchet, double_value


def main():
    worker = hatchet.worker("double-value-worker", slots=10)
    worker.register_workflow(double_value)
    worker.start()


if __name__ == "__main__":
    main()
